pages/common/ansible-pull.md

```diff
@@ -0,0 +1,20 @@
+# ansible-pull
+
+> Pull ansible playbooks from a VCS repo and executes them for the local host.
+> More information: <https://docs.ansible.com/ansible/latest/cli/ansible-pull.html>.
+
+- Pull playbook from a VCS and execute default local.yml playbook:
+
+`ansible-pull -U {{repository_url}}`
+
+- Pull playbook from a VCS and execute a specific playbook:
+
+`ansible-pull -U {{repository_url}} {{playbook}}`
+
+- Pull playbook from a VCS at specific branch and execute a specific playbook:
```

Comment:
Perhaps this will read better?
```suggestion
- Pull a playbook from a VCS at a specific branch and execute a specific playbook:
```

The missing `a` directly before `playbook` also applies to the other example descriptions too.

Reply:
Done!